tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -581,6 +581,27 @@ public function testDroppingConstraintsBeforeColumns(): void
         self::assertEquals($expectedSql, $sql);
     }
 
+    public function testDroppingPrimaryKey(): void
+    {
+        $oldTable = new Table('mytable');
+        $oldTable->addColumn('id', 'integer');
+        $oldTable->setPrimaryKey(['id']);
+
+        $newTable = clone $oldTable;
+        $newTable->dropPrimaryKey();
+
+        $diff = (new Comparator())->diffTable($oldTable, $newTable);
```

Comment:
The `diffTable()` method is deprecated. You should use `compareTables()` instead.